src/client/locale/I18.ts

```
@@ -0,0 +1,324 @@
+		if (!values)
+			return null;
+		if (values.length > 0 && Array.isArray(values[0]))
+			return values[0];
+		return values;
+	}*/
+
+	static getString(key: string, values?: string[] | null): string {
+		if (!values)
+			values = null;
+
+		const str = this.getStringOrNull(key, values);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)